tests/test_icml_conference.py

```diff
@@ -851,9 +876,15 @@ def test_ac_bidding(self, client, openreview_client, helpers, test_client):
             signatures=['~Program_ICMLChair1'],
             writers=[]
         ))
-        helpers.await_queue()        
+        helpers.await_queue()
         
-        assert openreview_client.get_invitation('ICML.cc/2023/Conference/Reviewers/-/Conflict')               
+        assert openreview_client.get_invitation('ICML.cc/2023/Conference/Reviewers/-/Conflict')
+
+        edges = openreview_client.get_edges(invitation='ICML.cc/2023/Conference/Reviewers/-/Conflict', head=submissions[0].id)
```

Comment:
could you check the length of affinity scores is `submissions * reviewers`?

I see the affinity score files is creating scores for all the notes (including desk-rejected).